lib/auth/apiserver.go

```diff
@@ -103,9 +102,11 @@ func NewAPIServer(config *APIConfig) http.Handler {
 	srv.POST("/:version/users/:user/ssh/authenticate", srv.withAuth(srv.authenticateSSHUser))
 	srv.GET("/:version/users/:user/web/sessions/:sid", srv.withAuth(srv.getWebSession))
 	srv.DELETE("/:version/users/:user/web/sessions/:sid", srv.withAuth(srv.deleteWebSession))
-	srv.GET("/:version/signuptokens/:token", srv.withAuth(srv.getSignupTokenData))
```

Comment:
I'm concerned about these. Why can't these be maintained for older proxies and but marked with `// DELETE IN: 4.4 (or 5.0)` so they can be removed in the next version of Teleport?

I understand you've done some refactoring around the internals, but can't the handler transform the old request to the new format?

Reply:
Old functionality requires most of the existing endpoints as the flow uses below methods as well.
```
CreateUserWithoutOTP
CreateUserWithOTP
CreateUserWithU2FToken
etc
```

My thinking was it's take too much time to ensure backward compatibility between old and new  signup UI flows. And since these are sensitive (security) changes I think it might be good to keep them to minimal.